hack/make/.integration-daemon-start

```diff
@@ -100,20 +103,19 @@ if [ -z "$DOCKER_TEST_HOST" ]; then
 		)
 	fi
 
-	# "pwd" tricks to make sure $DEST is an absolute path, not a relative one
```

Comment:
`$ABS_DEST` was right there :sob:  alas :heart:

Reply:
It was unexported, but it definitely wouldn't have been to hard to use it...